ci/src/cI_target.ml

```diff
@@ -87,6 +87,10 @@ let path = function
 
 type v = [ `PR of PR.t | `Ref of Ref.t ]
 
+let pp_v f = function
+  | `PR pr  -> Fmt.pf f "pr/%a" PR.pp pr
+  | `Ref x -> Fmt.pf f "refs/%a" Ref.pp x
```

Comment:
It seems strange that one case is plural and the other isn't.

Looks like we use `pr` and `ref` in URLs, but `prs/heads/tags` when parsing. Git uses `.git/refs`.